cinema/models.py

```diff
@@ -0,0 +1,8 @@
+from django.db import models
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField()
+    duration = models.IntegerField()
+    
```

Comment:
re-read the checklist.
Also, it's good practice to use the `str` method for models